lib/galaxy/config/__init__.py

```diff
@@ -1276,6 +1277,17 @@ def check(self):
                     f"Config option '{key}' is deprecated and will be removed in a future release.  Please consult the latest version of the sample configuration file."
                 )
 
+    def is_fetch_with_celery_enabled(self):
+        """
+        True iff celery is enabled and celery_conf["task_routes"]["galaxy.fetch_data"] == DISABLED_FLAG.
```

Comment:
@jdavcs Should this have been `celery_conf["task_routes"]["galaxy.fetch_data"] != DISABLED_FLAG` (or something like that)?

Reply:
I'll let you decide how to phrase it, if that's OK?